src/PKSim.UI/Views/ExpressionProfiles/CreateExpressionProfileView.cs

```diff
@@ -16,6 +17,9 @@ namespace PKSim.UI.Views.ExpressionProfiles
 {
    public partial class CreateExpressionProfileView : BaseModalView, ICreateExpressionProfileView
    {
+      private const int OPTIMAL_HEIGHT_NO_DISEASE_STATE = 220;
+      private const int OPTIMAL_HEIGHT_WITH_DISEASE_STATE = 270;
```

Comment:
Does this mean that disease states for now have 50px to work with? If we need more height, then adjust as needed?